### DISCUSSION TITLE
Auto-Update Powershell Script?

### DISCUSSION THREAD

**Discussion by mckenna654** [Eloston/ungoogled-chromium#3120] [Q&A]
Hi,

Just wondering if anyone has written an auto-update script for powershell for the windows version?

Thanks

**Comment by networkException** [comment]
Should be as simple as running `winget` update

**Comment by ArtefaCat** [comment]
Thank you! I also from the comment installed with a command via `winget` (`winget install ungoogled-chromium`) 
Had not used this tool before. Now your comment helped me to update:
`winget upgrade ungoogled-chromium`
